dcpandas/README.md

```diff
@@ -0,0 +1,47 @@
+# Data Commons Pandas API
+
+This is a Python library for creating pandas objects with data in the
+Data Commons Graph.
+To get started, install this package from pip.
+
+    pip install dcpandas
```

Comment:
I am debating on using datacomons-pd or datacommons-pandas. Using datacommons as prefix aligns with our normal "datacommns" package naming.

Reply:
I guess it's a bigger question we should talk about -- what are clients supposed to install and how will they interact with both libraries if they're doing a "standard" analysis using the library in pandas? Will they need to install both libraries and pick and choose what comes from dcpandas vs dc? Or will we continue wrapping up our other functions to be pandas friendly here (e.g. get_property_values, etc)

Reply:
So far as I am testing, I am installing both packages, so that I can do get_places_in, etc. Which is not so ideal.

I am not opposed to wrapping all functions, actually. But another alternative is to try to place the datacommons_pandas package inside the same directory, and having one setup.py ignore the pandas-related files. Otherwise the two setups are the same, and user can choose to import datacommons_pandas for the superset.

Reply:
That idea sgtm.

Reply:
I ended up placing datacommons and datacommons_pandas both at top level, but can certainly create another layer if that's better. I think this is fine, though. I also tried nesting datacommons_pandas inside datacommons, which did not work.

Reply:
I like the current solution!

Reply:
Glad you like it!